MapboxCoreNavigation/RouteController.swift

```diff
@@ -605,6 +630,91 @@ extension RouteController: CLLocationManagerDelegate {
         checkForFasterRoute(from: location)
     }
     
+    func checkForTunnelIntersection(at location: CLLocation, for manager: CLLocationManager, distanceTraveled: CLLocationDistance) {
+        let currentIntersection = routeProgress.currentLegProgress.currentStepProgress.currentIntersection
+        let shouldAnimate = shouldEnableTunnelAnimation(at: location, for: manager, intersection: currentIntersection)
+        
+        if shouldAnimate {
+            enableTunnelAnimation(for: manager,
+                        routeProgress: routeProgress,
+                     distanceTraveled: distanceTraveled)
+        } else {
+            suspendTunnelAnimation(for: manager, at: location)
+        }
+    }
+    
+    func shouldEnableTunnelAnimation(at location: CLLocation, for manager: CLLocationManager, intersection: Intersection?) -> Bool {
+        guard let currentIntersection = intersection else { return false }
+        
+        if let classes = currentIntersection.outletRoadClasses {
+            // Main conditions to enable simulated tunnel animation:
+            // - User location is within minimum tunnel entrance radius AND their location speed is at least 5 m/s
```

Comment:
It doesn't look like this speed check is still happening.

Reply:
Yes it is. 
https://github.com/mapbox/mapbox-navigation-ios/pull/1218/files#diff-1a2149692b3d7c58b6a0392120d6011aR765

Reply:
Okay, let's move the comment to there then.